tensorflow/tensorflow.bzl

```diff
@@ -316,29 +316,29 @@ def _cuda_copts():
     common_cuda_opts = ["-x", "cuda", "-DGOOGLE_CUDA=1"]
     return select({
         "//conditions:default": [],
-        "//third_party/gpus/cuda:using_nvcc": (
+        "@local_config_cuda//cuda:using_nvcc": (
             common_cuda_opts +
             [
                 "-nvcc_options=relaxed-constexpr",
```

Comment:
it would be nice if you could detect the version of `nvcc` and if is greater or equal to 7.5 then use `--expt-relaxed-constexpr`. unless we make 7.5 min requirement and just use that flag.  This will remove an annoying warning and all the tons of lines created inside logs.